pkg/rancher-desktop/backend/lima.ts

```diff
@@ -265,7 +265,10 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
 
   protected readonly CONFIG_PATH = path.join(paths.lima, '_config', `${ MACHINE_NAME }.yaml`);
 
-  protected cfg: BackendSettings | undefined;
+  /**
+   * Initialize this field to `defaultSettings` instead of `undefined` to avoid `?.` sequences everywhere we use it.
+   */
```

Comment:
But this means that we will no longer catch cases where we accidentally run with the default configuration instead of the user configuration (on startup).

Reply:
Reinstated